build/script/steamcmd

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [ `whoami` != "$STEAM_USER" ]; then
-	exec gosu $STEAM_USER $0 "$@"
-fi
-
```

Comment:
I think that `USER` only works inside the `Dockerfile` right? Anyway, I want my superuser with `docker exec`. This check can remove a lot of errors. Run this command as superuser can be difficult to fix.